tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist  = py35, py36, py37
+envlist  = py35, py36, py37, py38, py39
```

I think you forgot to remove 3.5
```suggestion
envlist  = py36, py37, py38, py39
```

---

Good catch.